### ISSUE TITLE
Customize Toolbar is disabled/gone

### ISSUE THREAD

**Issue by NSErfan** [ForkIssues/Tracker#1876]
Hi, I've noticed since [Fork 2.28](https://git-fork.com/releasenotes) (`Return 'Icon and Text' toolbar on macOS 13`) we cannot customize the toolbar.
- It's disabled in the Menu: `View>Customize Toolbar...`
<img width="375" alt="Screenshot 2023-04-25 at 6 35 30 PM" src="https://user-images.githubusercontent.com/3176129/234251648-fb933b4f-931f-4a61-8a7c-8b64f26f1c93.png">

- We used to be able to right click on toolbar and decide if we want to have `Icon Only`, `Text Only` and `Icon and Text`. 
_(the following screenshot is from Mail app)_
<img width="207" alt="Screenshot 2023-04-25 at 6 52 41 PM" src="https://user-images.githubusercontent.com/3176129/234255501-a8ebef22-4b56-4bf5-ae29-077dae5a8ab9.png">

This is a really useful feature for me, since I don't need the texts, icons are enough, And some of the buttons on the toolbars are not useful for me, by customizing toolbar I can keep the toolbar clean and focused on the things I actually used the most.

Thanks for developing this great app!

----

Environment:
- macOS 13.3.1 (22E261)

**Comment by DanPristupov** [comment]
https://github.com/fork-dev/Tracker/issues/1735
https://github.com/fork-dev/Tracker/issues/1774
https://github.com/fork-dev/Tracker/issues/1810

Yes, we disabled the customization.

**Comment by NSErfan** [comment]
Is this something temporary?  And once it's fixed it'll return?
I see that some other third-party apps have this feature and seems to be working fine on macOS 13.3.1.  ProxyMan for example which has a similar toolbar to Fork, too:
<img width="1330" alt="Screenshot 2023-04-26 at 9 53 42 AM" src="https://user-images.githubusercontent.com/3176129/234445782-e4e5e24d-8203-467c-89f9-8d0c151cb929.png">


It might be useful:
- I noticed they have put the capability to choose Icon Only, Text Only and Icon and Text in the app preferences:
<img width="601" alt="Screenshot 2023-04-26 at 9 55 53 AM" src="https://user-images.githubusercontent.com/3176129/234446117-e06b17e4-bb08-4933-9610-d37cdfcdfe38.png">

**Comment by username0x0a** [comment]
This change really is a bit unpleasant. 😕 I'd gladly spend my dev time on this to help you find a way to fix it if it's by any means possible, yet the app isn't open-source which makes it impossible atm. I know that `AppKit` is a wilderness on its own, but it's surely worth a try. ✊

**Comment by CodingMarkus** [comment]
I'm not using Ventura, I'm still using Monterey and I cannot customize the toolbar either. So even if the toolbar customization has issues in Ventura, why not disabling it only in Ventura? Nothing is easier than making code apply only to certain system versions (`@available` in Obj-C, `#available` in Swift; you can even use `__builtin_available` to get the same functionality in plain-C).

**Comment by NSErfan** [comment]
Hey @DanPristupov, this feature makes this daily tool (Fork) more pleasant. :) I hope you can consider re-enabling it somehow by gaining some inspiration from other third-party apps that have successfully managed this situation.

**Comment by homm** [comment]
Hi, Dan! Still not clear is this permanent? Still looking for a way to hide text from toolbar )

**Comment by DanPristupov** [comment]
Yes, most likely, yes. Currently we don't have plans (and time) to implement this kind of customization.

**Comment by BinaryHammer** [comment]
Simple: Without the option to disable the Feedback button, I will not buy it. Customizing the toolbar would solve my request.
